script.js

```diff
@@ -285,23 +285,19 @@
           const topicCell = $(this).find('td:eq(2)');
           const text = topicCell.text();
           const hue = mapTopicColor[text];
-          if (hue == null) {
-            topicCell.css('background-color', `black` );
-            return;
+          if (text != null && hue == null) {
+            addNewTopic(text);
           }
           topicCell.css('background-color', `hsl(${hue}, 70%, 81%)` );
       });
     }
 
     function addNewTopic(topic) {
-      if (lastestHue == null) {
-        lastestHue = 180;
-      }
-      mapTopicColor[topic] = lastestHue;
       lastestHue += 137;
       if (lastestHue > 360) {
```

Comment:
What if it's exactly 360? Do you want it to reset to 0 or stay as 360?

Using modulo operation is another solution.